src/main/java/seedu/address/model/person/Cap.java

```diff
@@ -36,7 +36,7 @@ public Cap(double cap, double max) {
      * @param max A maximum value for the CAP.
      */
     public static boolean isValidCap(double cap, double max) {
-        return cap <= max && cap >= MINIMUM;
+        return cap <= max && cap > MINIMUM && max <= 100.0;
```

Comment:
Should we update this limit into the UG as well?

Reply:
Will do, thanks for pointing that out!

Reply:
Pushed new commits to #258